storage/innobase/handler/ha_innodb.cc

```diff
@@ -17817,6 +17817,18 @@ ha_innobase::check_if_incompatible_data(
 	return(COMPATIBLE_DATA_YES);
 }
 
+bool ha_innobase::check_instant_alter(const Alter_inplace_info* inplace_info)
+{
+	if (inplace_info->handler_flags == Alter_inplace_info::ADD_STORED_BASE_COLUMN 
```

Comment:
The whole method should be removed, and the logic be implemented in `ha_innobase::check_if_supported_inplace_alter()` and `ha_innobase::prepare_inplace_alter_table()`.